pkg/gardenadm/cmd/token/create/options.go

```diff
@@ -6,17 +6,26 @@ package create
 
 import (
 	"fmt"
+	"regexp"
 	"strings"
 
 	"github.com/spf13/pflag"
 
 	"github.com/gardener/gardener/pkg/gardenadm/cmd"
+	tokenutils "github.com/gardener/gardener/pkg/gardenadm/cmd/token/utils"
+	"github.com/gardener/gardener/pkg/utils"
 )
 
+const charSet = "0123456789abcdefghijklmnopqrstuvwxyz"
```

Comment:
Can you replace the other occurrences of this literal with this constant as well?